user-guide/Basic_Functionality/Visio/SRM/Embedding_a_Resource_Manager_component.md

```diff
@@ -340,14 +340,22 @@ To also show resources for elements in child views, in the **ComponentOptions**
 > [!NOTE]
 > The corresponding resource bands are not updated automatically in case there is a change to the configuration of the elements.
 
-#### Passing exposers
+##### Passing exposers
 
-From DataMiner 10.3.1/10.4.0 onwards, you can pass exposers by object ID.
+From DataMiner 10.3.1/10.4.0 onwards, you can pass resource exposers as a filter by object ID.
 
-This enables a new filter option that can be used to show the corresponding resource bands on the timeline.
+This enables a new filter option that can be used to show the corresponding resource bands on the timeline. Using a filter which results in less than 100 resources is recommended to avoid delay while loading the bands.
+
+> [!TIP]
+> See also: [Filter feature examples](xref:YAxisResource_Shape_Data_Examples#examples-filter-feature)
+
+A converter has been created in the client which will convert a profile parameter name to the ID of the found object. For this, the format [ProfileParameter:xxx,ID] should be used, replacing 'xxx' with the name of the capacity/capability profile parameter.
 
 > [!NOTE]
-> The number of resources shown on a timeline is limited to 100 resources to avoid delay while loading the bands.
+> The server side filter is case sensitive with regard to profile parameter keys.
```

Comment:
Could you also add that it is expected that the profile parameter keys are represented with the number format.
Example:
- {d248b04f-2253-41c6-a92a-e12768fe2b20} is incorrect, this is format B (braces)
- (d248b04f-2253-41c6-a92a-e12768fe2b20) is incorrect, this is format P (parentheses)
- d248b04f-2253-41c6-a92a-e12768fe2b20 is correct, this is format D (default)
- d248b04f225341c6a92ae12768fe2b20 is correct, this is format N (number)

more info: https://learn.microsoft.com/en-us/dotnet/api/system.guid.tostring?view=netframework-4.8